DFC.Digital/DFC.Digital.Web.Sitefinity.CourseModule/Mvc/Views/CourseDetails/CourseDetails.cshtml

```diff
@@ -1,130 +1,196 @@
+                        @Model.CourseDetails.VenueDetails.Location.County<br />
+                        @Model.CourseDetails.VenueDetails.Location.Postcode<br />
+
+                        <a class="govuk-link" href="https://maps.google.com/?q=@Model.CourseDetails.VenueDetails.Location.Latitude,@Model.CourseDetails.VenueDetails.Location.Longitude">Find on Google maps</a>
                     </td>
                 </tr>
-            </tbody>
-        </table>
-    </div>
+                if (!string.IsNullOrEmpty(Model.CourseDetails.VenueDetails.Website))
+                {
+                    <tr class="govuk-table__row">
+                        <td class="govuk-table__cell govuk-body-s">Website</td>
+                        <td class="govuk-table__cell">
+                            <a class="govuk-link govuk-body-s" target="_blank" href="@Model.CourseDetails.VenueDetails.Website" style="word-break:break-all;">@Model.CourseDetails.VenueDetails.Website</a>
+                        </td>
+                    </tr>
+                }
+                if (!string.IsNullOrEmpty(Model.CourseDetails.VenueDetails.EmailAddress))
+                {
+                    <tr class="govuk-table__row">
+                        <td class="govuk-table__cell govuk-body-s">Email</td>
+                        <td class="govuk-table__cell govuk-body-s"><a href="mailto:@Model.CourseDetails.VenueDetails.EmailAddress" style="word-break:break-all;">@Model.CourseDetails.VenueDetails.EmailAddress</a> </td>
+                    </tr>
+                }
+
+                if (!string.IsNullOrEmpty(Model.CourseDetails.VenueDetails.PhoneNumber))
+                {
+                    <tr class="govuk-table__row">
+                        <td class="govuk-table__cell govuk-body-s">Phone</td>
+                        <td class="govuk-table__cell govuk-body-s">@Model.CourseDetails.VenueDetails.PhoneNumber</td>
+                    </tr>
+                }
 
+                if (!string.IsNullOrEmpty(Model.CourseDetails.VenueDetails.Fax))
+                {
+                    <tr class="govuk-table__row">
+                        <td class="govuk-table__cell govuk-body-s">Fax</td>
+                        <td class="govuk-table__cell govuk-body-s">@Model.CourseDetails.VenueDetails.Fax</td>
+                    </tr>
+                }
+            }
+        </tbody>
+    </table>
+
+
+    @Html.Partial("OtherDatesAndVenues", Model)
+
+
+    @if (!string.IsNullOrEmpty(Model.ReferralPath))
+    {
+        <a href="@Model.ReferralPath" class="govuk-back-link">Back to results</a>
+    }
+
+</div>
 
 <div class="govuk-grid-column-one-third">
-    @* RHC *@
 
     <aside class="app-related-items" role="complementary">
 
-        @Html.Partial("Provider", Model.ProviderDetails)
+        @Html.Partial("Provider", Model)
 
     </aside>
-</div>
\ No newline at end of file
+    <div class="app-related-items">
```

Comment:
Content authors should have access to change this content from CMS

Reply:
Added the contact us section as part of widget properties